### ISSUE TITLE
Hi! Will the dataset be available?

### ISSUE THREAD

**Issue by DLwbm123** [Tai-Hsien/PointNet_Seg#1]

**Comment by Tai-Hsien** [comment]
Hello,
Unfortunately, the dataset contains patients' data and is not public.

**Comment by DLwbm123** [comment]
Yeah, it's okay. I am searching for point cloud data in medical applications. ::)

**Comment by Tai-Hsien** [comment]
If it's hard to get point cloud in medical applications directly, you might go for some public medical database with volumetric images and labels, and then convert labels to mesh models (e.g., STL file) which is easy to transform to point cloud. :)

**Comment by DLwbm123** [comment]
Got it. Thanks for your kind advice and reply!

**Comment by Tai-Hsien** [comment]
You're very welcome! :)
